KomaMRICore/test/runtests.jl

```diff
@@ -120,7 +120,34 @@ using TestItems, TestItemRunner
         @test r1.Δf ≈ r2.Δf
         @test r1.delay ≈ r2.delay
     end
+
+    @testset "Delay" begin
+
+        # Test delay construction
+        T = 1e-3
+        delay = Delay(T)
+        @test delay.T ≈ T
+
+        # Test delay construction error for negative values
+        err = Nothing
+        try Delay(-T) catch err end
+        @test err isa ErrorException
+
+        # Test delay output message
```

Comment:
This is cool, but not very useful, if we change the way the Delay is printed it would throw an error.